src/current/_config_cockroachdb.yml

```diff
@@ -1,8 +1,8 @@
 baseurl: /docs
-current_cloud_date: '2023-08-03'
+current_cloud_date: '2023-09-26'
 current_cloud_version: v23.1
-current_dedicated_hotfix: v23.1.6
-current_serverless_hotfix: v23.1.6
+current_dedicated_hotfix: v23.1.10
+current_serverless_hotfix: v23.1.10
```

Comment:
nit: 
`current_serverless_hotfix` should be `v23.1.11`

Since on Tuesday 9/26, we'll be Upgrading serverless host clusters:

https://docs.google.com/document/d/1vWYuyYsuAzeelr9u6jAGVR7H1s9mr9jF9-xBe2CoEAE/edit

Reply:
Would it be okay if we put that in a separate PR after we first merge this PR? I imagine we're going to merge this PR at a point in time where v23.1.11 is only applicable to a select number of Dedicated customers.

Reply:
Confirmed over slack that this will be the case.